writeup/gz_hubble_data.tex

```diff
@@ -869,20 +869,38 @@ \section{The catalog}\label{sec:results}
+
+We use the complete bar sample to estimate the redshift evolution of bar fraction and find a steady decrease of $f_{bar} \sim 0.32$ at $z=0.4$ to $f_{bar} \sim 0.24$ at $z=1.0$. The decrease in bar fraction agrees with \ref{mel14}, however they report a lower bar fraction overall, finding a decrease from $f_{bar}=0.22$ at $z=0.4$ to $f_{bar}=0.11$ at $z=1.0$. The difference in total bar fraction is expected, as our analysis used a looser cut on $p_{\rm bar}$, we did not apply a luminosity cut, and we use debiased values for $p_{\rm features}$ which increases the total amount of disks in the sample. For a more thorough analysis of the bar fraction, a cut on $p_{\rm bar}$ that evolves with redshift may yield more accurate results, since there are no explicit debiased values of $p_{\rm bar}$ that would take redshift induced bias into account. 
+
+\subsection{Example 2: Identifying clump multiplicity}
+For galaxies identified as ``clumpy'' in GZH, the number of clumps can be determined using Task 16. We identify samples of galaxies having one, two, three, four, and more than four clumps. Using Table~\ref{tbl:thresholds}, we identify 8,444 clumpy galaxies using $p_{\rm features} > 0.23$ and $p_{\rm clumpy,yes}>0.80$ to ensure the vote fractions for Task 16 are well-sampled. We were able to identify with reasonable confidence the clump number of 1,112 of the clumpy galaxies; in the remainder, the unique clumps were less distinguishable from each other and the exact number of clumps could not be deduced without careful visual inspection. In the 1,112 which did have distinguishable clumps, we identified 61 one-clump galaxies using $p_{\rm 1~clump}>0.50$ 442 two-clump galaxies using $p_{\rm 2~clumps}>0.80$, 275 three-clump galaxies using $p_{\rm 3~clumps}>0.75$, 71 four-clump galaxies using $p_{\rm 4~clumps} > 0.70$, and 263 galaxies with more than four clumps using $p_{\rm >4~clumps}>0.70$. Alternatively, these data may be used to create more general samples of clumpy galaxies with few clumps and many clumps. We define a sample of 989 ``few clumps'' galaxies using $(p_{\rm 1~clump} + p_{\rm 2~clumps}) > 0.5$ and 2,910 ``many clumps'' galaxies using $(p_{\rm 3~clumps}+p_{\rm 4~clumps}+p_{\rm >4~clumps})>0.5$.
 
 {\note
 \begin{itemize}
-    \item explain decision tree format; vote fractions are dependent on previous task vote fractions
-    \item provide example of how to select bars
-    \item discuss difference between conservative/clean sample and loose/complete sample, emphasize goals differ depending on science case
+    \item \sout{explain decision tree format; vote fractions are dependent on previous task vote fractions} 
+    \item \sout{provide example of how to select bars}
+    \item \sout{discuss difference between conservative/clean sample and loose/complete sample, emphasize goals differ depending on science case}
+    \item \sout{science with bar example - estimate bar fraction vs redshift}
```

Comment:
Tom's PhD thesis has a nice description of this and also of how the fractions change when using the ferengi corrections...